pkg/commands/image/upload/upload.go

```diff
@@ -135,6 +136,7 @@ func UploadOstree(options UploadOptions) error {
 var providers map[string]func(UploadOptions) error = map[string]func(UploadOptions) error{
 	ProviderTypeOCI:    UploadOci,
```

Comment:
As a future enhancement, it would be good for oci images to be able to source their options from a cluster config as well.

Reply:
i will create jira